examples/walkthrough/k8s201.md

```diff
@@ -1,26 +1,69 @@
-# Kubernetes 201 - Labels, Replication Controllers, Services and  Health Checking
+# Kubernetes 201 - Labels, Replication Controllers, Services & Health Checking
 
-### Overview
-When we had just left off in the [previous episode](README.md) we had learned about pods, multiple containers and volumes.
-We'll now cover some slightly more advanced topics in Kubernetes, related to application productionization, deployment and
+If you went through [Kubernetes 101](README.md), you learned about kubectl, pods, volumes, and multiple containers.
+For Kubernetes 201 we'll cover some slightly more advanced topics in Kubernetes, related to application productionization, deployment and
```

Comment:
To clarify that the examples here build off of the 101 page, please insert/revise to:

For Kubernetes 201, we will pick up where 101 left off and cover....